jupyter_client/manager.py

```diff
@@ -29,13 +31,23 @@
     KernelManagerABC
 )
 
+class _ShutdownStatus(Enum):
+    Unset = None
+    ShutdownRequest = "ShutdownRequest"
+    SigtermRequest = "SigtermRequest"
+    SigKillRequest = "SigKillRequest"
```

Comment:
nit: Could you please either lowercase the 'K' in `SigKillRequest` or raise the 'T' in `SigtermRequest`?  I'd prefer the former.

Reply:
Done.